src/main/kotlin/com/nextstep/jngcii/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,25 @@
+package com.nextstep.jngcii.racingcar.domain
+
+class Car(
+    val name: String,
+    val speed: Int = SPEED,
+    val dice: Dice = Dice { true }
+) {
+    private var _distance = ZERO_DISTANCE
+    val distance get() = _distance
+
+    init {
+        require(name.isNotBlank()) { "자동차이름은 공백일 수 없습니다." }
+        require(name.length <= 5) { "자동차이름은 ${MAX_CAR_NAME_LENGTH}자를 초과할 수 없습니다." }
```

Comment:
```suggestion
        require(name.length <= MAX_CAR_NAME_LENGTH) { "자동차이름은 ${MAX_CAR_NAME_LENGTH}자를 초과할 수 없습니다." }
```
요기서도 정의해두신 상수를 사용하시면 더욱 좋을 것 같습니다 👍

Reply:
아고 놓쳤네요 ^^
[refactor : 상수 사용 누락 수정](https://github.com/next-step/kotlin-racingcar/pull/625/commits/c6263ad1b65c253ecad14002edc6050a83f23c02)
수정했습니다~!